app/models/applicant.rb

```diff
@@ -11,24 +11,6 @@ class Applicant < Person
     through: :crime_application
   )
 
```

Comment:
To allow code to be shared between datastore and database applications, most active record based scopes have been removed in favour of ruby. (See active model concerns PersonCapitalAssets for example.)